src/topp/MzTabExporter.cpp

```diff
@@ -782,8 +782,7 @@ namespace OpenMS
         db_version = sp.db_version.empty() ? MzTabString() : MzTabString(sp.db_version);
       }
 
-      // currently we don't save how many channels we saved in a consensus feature so we scan the file
-      // for the maximum map index in a consensus feature and take this as number of study variables
+      // determine number of channels
       Size n_study_variables = consensus_map.getFileDescriptions().size();
```

Comment:
The labels are also stored in the file descriptions, so there should be no problem. See http://ftp.mi.fu-berlin.de/pub/OpenMS/documentation/html/structOpenMS_1_1ConsensusMap_1_1FileDescription.html.